test/testcafe/spec/Identify_spec.js

```diff
@@ -238,6 +238,17 @@ test.requestHooks(identifyMock)('should render custom Unlock account link', asyn
   await t.expect(identityPage.getCustomUnlockAccountLink()).eql('http://unlockaccount');
 });
 
+test.requestHooks(identifyMock)('should not render custom forgot password link', async t => {
```

Comment:
should also add a happy path for other test ?

Reply:
It already exists: https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/spec/IdentifyWithPassword_spec.js#L54